scripts/sof-testbench-build-profile.sh

```diff
@@ -9,18 +9,28 @@ usage() {
     echo "  -h shows this text"
     echo "  -p <platform> sets platform for scripts/rebuild-testbench.sh"
     echo
+    echo "This script must be run from the sof firmware toplevel or workspace directory"
 }
 
 MODULES_S32="asrc dcblock drc drc_multiband eqfir eqiir gain src tdfb"
 MODULES_S24="aria"
 
-if [ -z "${SOF_WORKSPACE}" ]; then
-    echo "Error: environment variable SOF_WORKSPACE need to be set to top level sof directory"
+
+# check if we are in the sof FW directory
+if [ -f "Kconfig.sof" ]; then
```

Comment:
can we maybe use something like `$(dirname "$0")` to identify the target directory, where we have to switch to? Then it'll be possible to call the script from anywhere

Reply:
Ok, simplified this for non default installs - it will now check for ```SOF_WORKSPACE``` environment if set.